Terraform.gitignore

```diff
@@ -24,3 +24,6 @@ override.tf.json
 # Include override files you do wish to add to version control using negated pattern
 #
 # !example_override.tf
+
+# Ignore files created by `terraform plan -out=...`
+*plan
```

Comment:
Is this rule too vague? What's an example of the default file names here?

Reply:
There is no default value (plans are not saved by default).

But when developers do save plans, they usually (well, I noticed such usage and never met anything else) name the file "**plan**" or "blablabla.**plan**".